__tests__/spec/migrate.js

```diff
@@ -92,6 +107,11 @@ describe('migrate test prototype', () => {
     const layoutFileContents = fs.readFileSync(path.join(appDirectory, 'views', 'layout.html'), 'utf8')
 
     expect(layoutFileContents).toEqual(
+      '{#\n' +
+      'For documentation on layouts see:\n' +
```

Comment:
```suggestion
      'For guidance on how to use layouts see:\n' +
```

There is a new page for this.